Source/Core/Common/SysConf.h

```diff
@@ -44,15 +45,19 @@ struct SSysConfEntry
 	u8 nameLength;
 	char name[32];
 	u16 dataLength;
-	u8* data;
+	std::unique_ptr<u8[]> data;
```

Comment:
Is there a particular reason for not making this a std::vector? This is essentially what it's emulating.